examples/mysql/getting_started_step_3/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+#[macro_use] extern crate diesel;
+#[macro_use] extern crate diesel_codegen;
+extern crate dotenv;
+
+pub mod schema;
+pub mod models;
+
+use diesel::prelude::*;
+use diesel::mysql::MysqlConnection;
+use dotenv::dotenv;
+use std::env;
+
+use self::models::{Post, NewPost};
+
+pub fn establish_connection() -> MysqlConnection {
+    dotenv().ok();
+
+    let database_url = env::var("DATABASE_URL")
+        .expect("DATABASE_URL must be set");
+    MysqlConnection::establish(&database_url)
+        .expect(&format!("Error connecting to {}", database_url))
+}
+
+pub fn create_post(conn: &MysqlConnection, title: &str, body: &str) -> Post {
+    use schema::posts;
+
+    let new_post = NewPost {
+        title: title,
+        body: body,
+    };
+
+    diesel::insert(&new_post).into(posts::table)
+        .execute(conn)
+        .expect("Error saving new post");
+
+    posts::table.order(posts::id.desc()).first(conn).unwrap()
```

Comment:
If we `use schema::posts::dsl::*` then we need to change the `body` and `title` variable names;

I added `use schema::posts::dsl::{posts, id};` instead